src/main/java/org/folio/dew/batch/bursarfeesfines/service/BursarTokenFormatter.java

```diff
@@ -355,7 +355,7 @@ public static String formatAggregatedAccountsToken(
     }
   }
 
-  public static String applyLengthControl(
```

Comment:
Any particular reason this got moved from public to private?  IMO it should be left public, if for no other reason than to aid unit tests

Reply:
I just assumed it was an oversight since the method wasn't being used outside of the `BursarTokenFormatter` class. Will revert to `public`!